statsd/tests/test_statsd.py

```diff
@@ -41,16 +46,24 @@ def get_instance():
 @pytest.fixture(scope='session', autouse=True)
 def spin_up_statsd():
     with docker_run(
-        compose_file=os.path.join(HERE, 'compose', 'statsd.yaml'),
-        endpoints=URL
+        compose_file=os.path.join(HERE, 'compose', 'statsd.yaml')
     ):
+
+        # In Python 3 the url checker does not work.
+        # It doesn't handle raw tcp connections
+        # in fact the only reason it worked in Python 2 was that it was broken
+        # The only good way to mock this would be to duplicate the logic in the check
+        # which doesn't make a very good test.
+        # Instead, just add a sleep here since the container is pretty quick to spin up
+        time.sleep(10)
```

Comment:
```suggestion
        log_patterns=['server is up']
```

Reply:
Looks like the container emits the above log line that we can wait for instead of hardcoding a time.